config/config.go

```diff
@@ -39,6 +39,9 @@ type MongoConfig struct {
 	OutputsCollection string `envconfig:"MONGODB_OUTPUT_COLLECTION"`
 	Limit             int    `envconfig:"MONGODB_LIMIT"`
 	Offset            int    `envconfig:"MONGODB_OFFSET"`
+	Username          string `envconfig:"MONGO_USERNAME"`
```

Comment:
Please ensure to add relevant dp config entries for update username, password and mongodb bind address.